shell/platform/fuchsia/flutter/default_session_connection.cc

```diff
@@ -316,6 +366,15 @@ void DefaultSessionConnection::PresentSession() {
 
   last_latch_point_targeted_ = next_latch_point;
 
+  // Flutter is presenting a frame here, so mark it as such.
+  // Inspect updates must run on the inspect dispatcher.
+  async::PostTask(
+      inspect_dispatcher_,
+      [this, now = fml::TimePoint::Now().ToEpochDelta().ToNanoseconds()]() {
+        frames_presented_.Add(1);
```

Comment:
I'd change this to presents_submitted, because frames_presented can be confused with the OnFramesPresented() callback that fires :)

Reply:
Done